astropy/cosmology/funcs/comparison.py

```diff
@@ -8,24 +8,24 @@
 
 from __future__ import annotations
 
+__all__ = []  # Nothing is scoped here
+
```

Comment:
[This change does make the file comply with PEP 8](https://peps.python.org/pep-0008/#module-level-dunder-names), but it's not related to dropping support for Python 3.9, so in this pull request it's just adding clutter.

Reply:
✅